pkg/parsing/sql/sql.go

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2018 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package sql provides a basic SQL SQLParser that can be extended by individual implementations.
+// Because Trickster is a proxy/cache for time series, this package currently only has base
+// support for specific SELECT statements
+//
+// NOTE: While Trickster does not proved a true, full AST; we would love to and welcome all contributions
+//
+package sql
+
+import (
+	"context"
+	"errors"
+
+	"github.com/tricksterproxy/trickster/pkg/parsing"
+	lsql "github.com/tricksterproxy/trickster/pkg/parsing/lex/sql"
+	"github.com/tricksterproxy/trickster/pkg/parsing/token"
+)
+
+// ErrUnsupportedVerb represents an error of type "unsupported verb"
+var ErrUnsupportedVerb = errors.New("unsupported verb")
+
+// ErrUnsupportedClause represents an error of type "unsupported clause"
+var ErrUnsupportedClause = errors.New("unsupported clause")
+
+// // Parser defines the SQL Parser Interface
+// type Parser interface {
```

Comment:
ded?

Reply:
will remove this one since it was relocated to timeseries/sqlparser